cmd/earthly/account_cmds.go

```diff
@@ -243,7 +243,7 @@ func (app *earthlyApp) actionAccountRegister(cliCtx *cli.Context) error {
 	var publicKey string
 	if app.registrationPublicKey == "" {
 		if len(publicKeys) > 0 {
-			rawIsRegisterSSHKey, err := promptInput(cliCtx.Context, "Would you like to register an SSH key to be used as a form of login? [Y/n]: ")
+			rawIsRegisterSSHKey, err := promptInput(cliCtx.Context, "Would you like to enable password-less login using public key authentication (https://earthly.dev/public-key-auth)? [Y/n]: ")
```

Comment:
NOTE: we'll need to create a new page that describes how this works in greater detail;

at first I tried explaining it completely on stdout, but it made following the wizard/interactive-prompt quite difficult:

here's the initial version I tried:

```
New password: 
Confirm password: 
I acknowledge Earthly Technologies' Privacy Policy (https://earthly.dev/privacy-policy) and agree to Earthly Technologies Terms of Service (https://earthly.dev/tos) [y/N]: y
Earthly supports password-less login using public key authentication.
To enable public key authentication, your public key must be stored in an authorized_keys database on the earthly server. (earthly will never read (or transmit) your private key, all authentication-challenges are handled by the ssh-agent process)

Would you like to associate a public key with your account? [Y/n]: 
```

If we really want to describe it on the command line, then perhaps we need to add some newlines between all sections, e.g.

```
New password: 
Confirm password: 

I acknowledge Earthly Technologies' Privacy Policy (https://earthly.dev/privacy-policy) and agree to Earthly Technologies Terms of Service (https://earthly.dev/tos) [y/N]: y

Earthly supports password-less login using public key authentication. To enable public key authentication, your public key must be stored in an authorized_keys database on the earthly server. (earthly will never read (or transmit) your private key, all authentication-challenges are handled by the ssh-agent process)
Would you like to associate a public key with your account? [Y/n]: 
```